packaging/docker/entrypoint.py

```diff
@@ -192,9 +206,11 @@ def load_config() -> tuple[list[str], str | None]:
     '0.0.0.0',
 ]
 
-cmd = base_args + config_args
+# Arguments needs to be sanitized for `subprocess.Popen`, as it does not accept `int`
+cmd = sanitize_args_for_popen(base_args + config_args)
 
 logger.info('starting rotki backend')
+logger.debug('backend arguments: %s', cmd)
```

Comment:
Keep a single log entry for this

```suggestion
logger.info(f'starting rotki backend with args: {cmd}')
```